ql/src/java/org/apache/hadoop/hive/ql/ddl/table/partition/PartitionUtils.java

```diff
@@ -61,25 +61,37 @@ private PartitionUtils() {
    * with these reserved values. The check that this function is more restrictive than the actual limitation, but it's
    * simpler. Should be okay since the reserved names are fairly long and uncommon.
    */
-  public static void validatePartitions(HiveConf conf, Map<String, String> partitionSpec) throws SemanticException {
-    Set<String> reservedPartitionValues = new HashSet<>();
+  public static void validatePartitions(HiveConf conf, Map<String, String> partitionSpec) {
```

Comment:
please remove 2 `are` from the java doc in "Certain partition values are are used by hive"

Reply:
Sure